test/setup/setup.spec.js

```diff
@@ -0,0 +1,5 @@
+describe('setup', () => {
```

Comment:
I'm not sure if this file is necessary. We're leaning towards a "minimal configuration" with regard to the boilerplate's setup